Toolset/palettes/message box/behaviors/revmessageboxbehavior.livecodescript

```diff
@@ -98,6 +96,38 @@ on openStack
    unlock screen
 end openStack
 
+on ideMsgChanged pTarget, pHandler, pLine, pMsg
```

Comment:
Ah in that case I think it is missing from https://github.com/livecode/livecode-ide/pull/1537/files#diff-96553dde6940d369c385ef6d37a9405eR407